packages/closer/components/PageEditor/inspectors/MediaInspector.tsx

```
@@ -0,0 +1,99 @@
+import { useTranslations } from 'next-intl';
+
+import { Input, Textarea } from '../../ui';
```

ESLint

[error] 'Textarea' is defined but never used.

(unused-imports/no-unused-imports)